apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -1,19 +1,47 @@
 import React, {PropTypes} from 'react';
-import {Button, FormControl} from 'react-bootstrap';
-import {Facilitator1819Program} from './detail_view_facilitator_specific_components';
+import {Button, FormControl, Panel} from 'react-bootstrap';
+import Facilitator1819Questions from './detail_view_facilitator_specific_components';
 import $ from 'jquery';
+import _ from 'lodash';
+
+// Public function to render a question/answer pair as either a line item or a panel
+// depending on whether the question will be assigned score
+const renderItem = (key, value, layout, iteratorKey) => {
+  if (typeof value === 'string' || typeof value === 'object') {
+    let renderedValue = value;
+    if (Array.isArray(value)) {
+      renderedValue = _.join(value, ', ');
+    }
 
-const renderLineItem = (key, value) => {
+    if (layout === 'lineItem') {
+      return renderLineItem(key, renderedValue, iteratorKey);
+    } else {
+      return renderQuestionBox(key, renderedValue, iteratorKey);
+    }
+  }
+};
+
+const renderLineItem = (key, value, iteratorKey) => {
+  // If there is a value to this line item, render the question. Then optionally
+  // render a : if the question does not end in ?, :, or . Then render the answer.
   return value && (
-      <div>
-      <span style={{fontFamily: '"Gotham 7r"', marginRight: '10px'}}>
-        {key}:
-      </span>
+      <div key={iteratorKey}>
+        <span style={{fontFamily: '"Gotham 7r"', marginRight: '10px'}}>
+          {`${key}${'?:.'.indexOf(key[key.length - 1]) >= 0 ? '' : ':'}`}
+        </span>
         {value}
       </div>
     );
 };
 
+const renderQuestionBox = (key, value, iteratorKey) => {
+  return value && (
+    <Panel key={iteratorKey} header={key} style={{display: 'table'}}>
+      {value}
+    </Panel>
+  );
+};
```

Comment:
Same with these. I'd rather see this contained in the above `QuestionAnswer` component (or potentially in their own subcomponents, though these don't appear to be used on their own).